src/curated-corpus/components/ScheduleItemFormConnector/ScheduleItemFormConnector.test.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MockedProvider } from '@apollo/client/testing';
+import { SnackbarProvider } from 'notistack';
+import { ScheduleItemFormConnector } from './ScheduleItemFormConnector';
+import LuxonUtils from '@date-io/luxon';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import {
+  mock_AllScheduledSurfaces,
+  mock_OneScheduledSurface,
+  mock_TwoScheduledSurfaces,
+} from '../../integration-test-mocks/getScheduledSurfacesForUser';
+import userEvent from '@testing-library/user-event';
+import { mock_ScheduledItemCountsZero } from '../../integration-test-mocks/getScheduledItemCounts';
+import { DateTime } from 'luxon';
+
+describe('ScheduleItemFormConnector', () => {
+  let mocks = [];
+
+  it('loads the form with all scheduled surfaces', async () => {
+    mocks = [mock_AllScheduledSurfaces];
+
+    render(
+      <MockedProvider mocks={mocks}>
```

Comment:
was poking around in [the apollo docs](https://www.apollographql.com/docs/react/development-testing/testing/#setting-addtypename) this morning and saw they recommend adding `addTypename={false}` to these `<MockedProvider>` components. these tests seem to be working fine as-is though, so...?